ui/pages/providers/index.js

```diff
@@ -92,25 +97,96 @@ export default function Providers() {
             accessorKey: 'clientID',
           },
           {
+            id: 'actions',
             cell: function Cell(info) {
-              const [open, setOpen] = useState(false)
+              const [deleteOpen, setDeleteOpen] = useState(false)
+              let [referenceElement, setReferenceElement] = useState()
```

Comment:
can we change it to `const [referenceElement, setReferenceElement] = useState(null)` ?